cmd/cloud/installation_flag.go

```diff
@@ -113,6 +158,25 @@ func (flags *installationUpdateFlags) addFlags(command *cobra.Command) {
 	_ = command.MarkFlagRequired("installation")
 }
 
+func (flags *installationUpdateFlags) GetPatchInstallationRequest() (*model.PatchInstallationRequest, error) {
+	request := flags.installationPatchRequestOptions.GetPatchInstallationRequest()
+
+	envVarMap, err := parseEnvVarInput(flags.mattermostEnv, flags.mattermostEnvClear)
```

Comment:
Can we make it `mattermostEnv` to match it with `priorityEnv`?

Reply:
Oh. I mean the variable name. `envVarMap` -> `mattermostEnv `

Reply:
Oh, I was wondering if you referred to the logic itself. Yeah, that make sense.